stories/atoms/List.stories.tsx

```diff
@@ -60,8 +59,6 @@ export const _List: Story = {
                 <Checkbox id="item4" />
                 Clickable item
             </ListItem>,
-            <SingleSelectItem>Suggestion Item</SingleSelectItem>,
-            <SingleSelectItem isSelected>Suggestion Item Selected</SingleSelectItem>,
         ],
     },
     render: (args) => <List {...args} />,
```

Comment:
cannot make a comment in the right place, but can you remove https://oneui-git-oneui-496-refactor-68571a-textkernel-oneuis-projects.vercel.app/?path=/story/atoms-list--list-with-separators&vercelToolbarCode=H8DlhrV7xZ6JF0r (separators story)